test/README.md

```diff
@@ -33,36 +37,47 @@ these steps:
   2. Copy and/or rename the `tmp/server/debug/rippled.cfg` file to somewhere
   convenient.
 
-4. Start `rippled` (in a debugger) with command line options
+##### Using the command line
+
+1. Create a `rippled.cfg` file for the tests.
+  1. Run `npm test --noserver`. The tests will fail. **This failure is expected.**
```

Comment:
I thought I saw `This failure is expected.` before?!  But it seems to be new in this change?  Baffling!

Thanks for adding documentation!!